plugins/workspace-backpack/src/ui_events.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview UI events used for Backpack.
+ * @author kozbial@google.com (Monica Kozbial)
+ */
+
+import * as Blockly from 'blockly/core';
+
+/**
+ * Name of event that records a backpack open.
+ * @const
+ */
+export const BACKPACK_OPEN = 'backpack_open';
+
+/**
+ * A UI event representing a change in the Backpack contents.
+ */
+export class BackpackOpen extends Blockly.Events.UiBase {
+  /**
+   * Type of this event.
+   * @type {string}
+   */
+  type = BACKPACK_OPEN;
+
+  /**
+   * Class for a backpack open event.
+   * @param {boolean=} isOpen Whether the backpack flyout is opening (false
```

Comment:
Here it says false if opening, but in `this.isOpen` it says false if closing. 

Maybe `isOpening` would be more clear if the backpack is opening during this event vs was already open?

Reply:
It's a typo that came from copying it from `Blockly Events.TrashcanOpen`. It should be "false if closing".
I'd refer to leave it as `isOpen` for consistency with the `TrashcanOpen` event.